spec/faraday/adapter_registry_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+RSpec.describe Faraday::AdapterRegistry do
+  describe '.initialize' do
```

Comment:
nit: although called during initialisation, `initialize` is still an instance method, so it should be prefixed with `#` instead of `.` (the latter is for class methods)